docs/specs/om/open_metrics_spec_1_1.md

```diff
@@ -126,6 +128,8 @@ MetricFamily names beginning with underscores are RESERVED and MUST NOT be used
 
 The name of a MetricFamily MUST NOT result in a potential clash for sample metric names as per the ABNF with another MetricFamily in the Text Format within a MetricSet. An example would be a gauge called "foo_created" as a counter called "foo" could create a "foo_created" in the text format.
 
+Ingestors MAY accept duplicate MetricFamily names, provided that either of HELP, TYPE, or UNIT is unique within the duplicated MetricFamily names. Notably Exposers MAY duplicate MetricFamilies when aggregating external metrics that have the same MetricFamily name, but come from different sources and have either different HELP, or TYPE, or UNIT. Be aware that exposing such metrics directly to end-users reduces usability due to confusion about which Metric belongs to which MetricFamily.
```

Comment:
I was aware of type and unit being identified, but this is the first I'm hearing of HELP being identifying -- is that a case we need to support?

Reply:
let's decide https://github.com/prometheus/docs/pull/2745#discussion_r2399091513 first